features/languages-testcase.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Languages
+
+Scenario Outline: User is able to change the language
+           Given User sees <country-flag> flag on the pah-fm website
+            When User clicks on <country-flag> flag
+            Then The language on the page changes to <chosen-language>
+
+  Examples:
+| country-flag   | chosen-language |
+| Ukraine        | Ukrainian       |
+| United Kingdom | English         |
+| Poland         | Polish          |
+
+Scenario Outline: Language is retained after attempt to login
+            Given User is on the pah-fm website
+              And <chosen-language> has been chosen
+            When User log in <status>
+            Then The language is still <outcome-language>
+
+  Examples:
+| chosen-language | outcome-language | status         |
```

Comment:
I've also used to indent the tables (by 2 or 4 spaces) - it's up to you to decide (or check with official documentation).

Reply:
That applies to other tables as well.

Reply:
Indented by 2 spaces as suggested (also checked Behave documentation)